packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_ExpressionValue.res

```diff
@@ -5,26 +5,33 @@
 module Extra_Array = Reducer_Extra_Array
 module ErrorValue = Reducer_ErrorValue
 
+@genType.opaque
+type internalCode = Object
+
 @genType
 type rec expressionValue =
   | EvArray(array<expressionValue>)
   | EvBool(bool)
   | EvCall(string) // External function call
   | EvDistribution(DistributionTypes.genericDist)
+  | EvLambda((array<string>, internalCode))
   | EvNumber(float)
   | EvRecord(Js.Dict.t<expressionValue>)
   | EvString(string)
   | EvSymbol(string)
 
 @genType
 type externalBindings = Js.Dict.t<expressionValue>
+@genType
+let defaultExternalBindings: externalBindings = Js.Dict.empty()
 
 type functionCall = (string, array<expressionValue>)
 
 let rec toString = aValue =>
   switch aValue {
   | EvBool(aBool) => Js.String.make(aBool)
   | EvCall(fName) => `:${fName}`
+  | EvLambda((parameters, _internalCode)) => `lambda(${Js.Array2.toString(parameters)}=>internal)`
```

Comment:
shouldn't this be `=>_internalCode`?

Reply:
Internal code is for externalBindings. Not inside Reducer

Reply:
Oh I see that it says `internal` rather than `internalCode`, but still--- wouldn't `internal` be `_internalCode`, and if not, `_internalCode` should be an underscore.

Reply:
castInternalCode or asInternalCode would be suitable instead of just internal

Reply:
compiled form of , for example
\x -> 2*x
is shown as 
lambda(x)=>internal
Personally, I don't think it is not obscure.
This is a toString format. It is not a call.
It could be printed as 
lambda(x)=>internal code
But that would take more screen space on screen test scripts. Nobody will show those outside testing.